ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -156,28 +156,12 @@ Item {
         }
     }
 
-    StackLayout {
+    Item {
```

Comment:
The Item shouldn't be needed anymore. I had kept it just because I wanted to test quickly